src/MessagePack.UnityClient/Assets/MessagePack.UnityClient.Dependencies.msb4u.csproj

```diff
@@ -0,0 +1,42 @@
+<Project ToolsVersion="15.0">
+  <!--GENERATED FILE-->
+  <!--
+    This file can be modified and checked in.
+
+    It is different from the other generated C# Projects in that it will be the one gathering all dependencies and placing them into the Unity asset folder.
+
+    You can add project level dependencies to this file, by placing them below:
+    - <Import Project="$(MSBuildForUnityGeneratedProjectDirectory)\$(MSBuildProjectName).g.props" />
+    and before:
+    - <Import Project="$(MSBuildForUnityGeneratedProjectDirectory)\$(MSBuildProjectName).g.targets" />
+
+    Do not add any source or compilation items.
+
+    Examples of how you can modify this file:
+    - Add NuGet package references:
+        <ItemGroup>
+          <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
+        </ItemGroup>
+    - Add external C# project references:
+      <ItemGroup>
+        <ProjectReference Include="..\..\..\ExternalLib\ExternalLib.csproj" />
+      </ItemGroup>
+  -->
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(MSBuildForUnity.Common.props))" Condition="Exists('$([MSBuild]::GetPathOfFileAbove(MSBuildForUnity.Common.props))')" />
```

Comment:
I didn't realize you could get away with not surrounding filenames in single-quotes like this. TIL.